doc/Caliptra.md

```diff
@@ -1332,64 +1332,62 @@ The following table describes Caliptra's fuse map:
 | LMS VERIFY                      | 1               | ROM             | In-field programmable                           | <p>0 - Verify Caliptra firmware images with ECDSA-only.</p><p>1 - Verify Caliptra firmware images with both ECDSA and LMS.</p>
 | LMS REVOCATION                  | 32              | ROM             | In-field programmable                           | Bits for revoking LMS public keys in the key manifest.
 
-# Error Reporting and Handling
+# Error reporting and handling
 
 This section describes Caliptra error reporting and handling.
 
-*Table 14: HW/FW Error Types*
+*Table 14: Hardware and firmware error types*
 
-| | Fatal Errors | Non-fatal Errors |
+| | Fatal errors | Non-fatal errors |
 | :- | - | - |
-| HW | <p>- ICCM, DCCM SRAM ECC</p><p>- Second WD timer expiry (First one triggers an NMI to FW to correct the issue and clear the interrupt status bit. If the WD expires again, then it gets escalated to FATAL error</p> | <p>- Mailbox SRAM ECC (except initial FW load)</p><p>- Mailbox incorrect protocol/commands. E.g. Incorrect access ordering; access without Lock.</p> |
-| FW | <p>- Boot-time FW authentication failures</p><p>- FW triggered FATAL errors. Examples: ICCM/DCCM misaligned access (potentially in the except subroutine); AHB access hangs that is triggered through WD timer expiry; AHB access outside the decoding range; stack overflow etc. Refer to the table below</p> | <p>- Mailbox API failures</p><p>- First WD timer expiry</p><p>- Crypto processing errors</p> |
+| Hardware | <p>- ICCM, DCCM SRAM ECC</p><p>- Second WD timer expiry. The first timer expiry triggers an NMI to firmware to correct the issue and clear the interrupt status bit. If the WD expires again, then it is escalated to a FATAL error.</p> | <p>- Mailbox SRAM ECC (except initial firmware load)</p><p>- Mailbox incorrect protocol or commands. For example, incorrect access ordering or access without Lock.</p> |
```

Comment:
`WD` needs to be defined or spelled out as `watchdog`.

Reply:
updated